modules/globebrowsing/src/dashboarditemglobelocation.cpp

```diff
@@ -57,6 +57,12 @@ namespace {
         openspace::properties::Property::Visibility::AdvancedUser
     };
 
+    // This `DashboardItem` shows the longitude/latitude location of the camera and its
+    // distance to the current focus node. If the current focus node is Earth, these are
+    // provided in the WGS84 reference frame, if the focus is on another planetary body,
```

Comment:
```suggestion
    // provided in the WGS84 reference frame. If the focus is on another planetary body,
```

Reply:
Replaced it with a semicolon since I want to keep the dichotomy between the If Earth / If something else